modules/core/src/main/java/org/apache/ignite/internal/processors/cache/query/GridCacheQueryManager.java

```diff
@@ -2090,6 +2090,12 @@ public String space() {
         /** */
         private static final long serialVersionUID = 0L;
 
+        /**
+         * Number of fields to report when no fields defined.
+         * Includes _key and _val columns.
+         */
+        private static final int DEFAULT_FIELDS_COUNT = 2;
```

Comment:
We have another constant `DEFAULT_COLUMNS_COUNT` which is `3` but not `2`. They are hard to distinguish. Lets rename this one to `NO_FIELDS_COLUMNS_COUNT`.